CITATION.cff

```diff
@@ -0,0 +1,26 @@
+cff-version: 1.1.0
+authors:
+  - family-names: Shapiro
+    given-names: Joshua A.
+    orcid: https://orcid.org/0000-0002-6224-0347
+  - family-names: Savonen
+    given-names: Candace L.
+    orcid: https://orcid.org/0000-0001-6331-7070
+  - family-names: Hawkins
+    given-names: Allegra
+    orcid: https://orcid.org/
+  - family-names: Bethell
+    given-names: Chante J.
+    orcid: https://orcid.org/0000-0001-9653-8128
+  - family-names: Venkatesh Prasad
+    given-names: Deepashree
+    orcid: https://orcid.org/0000-0001-5756-4083
+  - family-names: Greene
+    given-names: Casey S.
+    orcid: https://orcid.org/0000-0001-8713-9213
+  - family-names: Taroni
+    given-names: Jaclyn N.
+    orcid: https://orcid.org/0000-0003-4734-4508
+title: "Childhood Cancer Data Lab"
```

Comment:
Do we want to be a bit more specific? (I do not know if we are using title or sentence case)
```suggestion
title: "Childhood Cancer Data Lab genomics training materials"
```